### ISSUE TITLE
Image with alt text - Custom alt text removed when adding an image from media library that has no alt text

### ISSUE THREAD

**Issue by andeng1106** [gambitph/Stackable#2524]
**Describe the bug**
When I added an image from media library that has no alt text, custom alt text added in block inspector was being removed.

Happens to blocks that has images with alt text including but not limited to:
- Image block (feature/settings)
- Card block (feature/settings)
- Video Popup (nested image block)
- Image Box (nested image block)
- Feature (nested image block)
- Feature Grid (nested image block)
- Hero (nested image block)
- Team Member (nested image block)
- Testimonial (nested image block)

**To Reproduce**
1. Add an Image block or Card block
2. Add a custom alt text in block inspector (IMAGE ALT field)
3. Add an image from media library that has no alt text
4. Check IMAGE ALT field if custom alt text disappeared

**Expected behavior**
- Scenario/s should be (to confirm with Jami):
  - If uploaded media library image has existing alt text, should overwrite custom alt text
  - If uploaded media library image has no alt text (empty or blank), should NOT overwrite custom alt text

**Screenshots**
**Card block**

https://user-images.githubusercontent.com/103395655/203734573-e0cb80be-308d-4e89-904e-b65b1e4e3111.mp4

**Image block**

https://user-images.githubusercontent.com/103395655/203734936-926d3fba-eb74-4db7-ad53-6d2f9236993e.mp4
